lib/profile/common/styles.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+TextStyle fieldTextStyle = GoogleFonts.muli(
+  textStyle: TextStyle(
+    color: Colors.black, 
+    letterSpacing: .5, 
+    fontSize: 13.0, 
+  )
+);
+
+TextStyle buttonTextStyle = GoogleFonts.muli(
+  textStyle: TextStyle(
+    color: Colors.white, 
+    letterSpacing: .5, 
+    fontSize: 13.0, 
+    fontWeight: FontWeight.w700
+  )
+);
+
+TextStyle labelTextStyle({bool isEnabled}) => GoogleFonts.muli(
+  textStyle: TextStyle(
+    color: isEnabled ? Colors.black : Colors.black38, 
+    letterSpacing: .5, 
+    fontSize: 13.0, 
+  )
+);
+
+TextStyle pickerTextStyle({bool isCancel, bool isButton}) => GoogleFonts.muli(
+  textStyle: TextStyle(
+    color: isCancel ? Colors.red : Colors.black, 
+    letterSpacing: .5, 
+    fontSize: 15.0,
+    fontWeight: isButton ? FontWeight.w600 : FontWeight.normal
+  )
+);
+
+InputDecoration fieldDecoration({String errorText}) => InputDecoration(
+  isDense: true,
+  contentPadding: EdgeInsets.only(
+    top: 5.0,
+    bottom: 5.0
+  ),
+  errorText: errorText
```

Comment:
Is it not like ES6 where you can pass `errorText` as the key and value?

Reply:
Oooo, not sure what you mean... Send a link! I just followed `InputDecoration`'s API here: 
https://api.flutter.dev/flutter/material/InputDecoration-class.html
https://api.flutter.dev/flutter/material/InputDecoration/errorText.html

Reply:
I think Helen means your name it as aB: aB which is the same.

Reply:
Oh, I see what you mean now! Yeah, that's actually what I initially tried but it gave me an error. I don't think ES6 works on Dart... ?

<img width="541" alt="Screen Shot 2020-06-30 at 11 07 16 AM" src="https://user-images.githubusercontent.com/25041263/86155603-e44e1b80-bac1-11ea-81c6-9f4195584278.png">